FinancialTransactionsApi/appsettings.json

```diff
@@ -32,5 +32,11 @@
       "UseLegacyPorts": false,
       "EdgePort": 4566
     }
+  },
+  "ReportExportSettings": {
```

Comment:
Can this be added to env vars (e.g. add in parameter store for now and retrieve/set as env var in serverless.yml) to avoid need of redeployment?

Reply:
Done